treeshr/TreeGetRecord.c

```diff
@@ -136,22 +138,16 @@ int TreeGetRecord(int nid_in, mdsdsc_xd_t *dsc)
                 status = TreeGetExtendedAttributes(
                     info, RfaToSeek(nci.DATA_INFO.DATA_LOCATION.rfa),
                     &attributes);
-                if (STATUS_OK &&
-                    attributes
-                            .facility_offset[STANDARD_RECORD_FACILITY] !=
-                        -1)
+                RETURN_IF_STATUS_NOT_OK;
```

Comment:
If `TreeGetExtendedAttributes()` encountered errors, this line probably returns a status of `TreeFAILURE` whereas the original code likely returned `TreeBADRECORD`.   However, the revised `TreeSegmentTest.c` works fine, thus presumably the different status code returned isn't important.

Reply:
replaced it with status = TreeBadRecord. this also keeps the program sequence uniform (single return statement)